tendrl/commons/objects/cluster/atoms/configure_monitoring/__init__.py

```diff
@@ -157,7 +157,8 @@ def run(self):
         _cluster = NS.tendrl.objects.Cluster(
             integration_id=NS.tendrl_context.integration_id
         ).load()
-        _cluster.import_status = "done"
+        _cluster.status = ""
+        _cluster.current_job['status'] = "done"
```

Comment:
we have been using "finished" all over Tendrl, lets stick to that instead of "done"

Reply:
done